docs/src/content/docs/reference/configuration.md

```diff
@@ -317,3 +317,12 @@ interface HeadConfig {
   content?: string;
 }
 ```
+
+### `lastUpdated`
+
+**type:** `boolean`  
+**default:** `false`
+
+Define if the last update date should be visible in the page footer.
```

Comment:
```suggestion
Control whether the footer shows when the page was last updated.
```
I kind of like something like this! It makes it explicit that the display will ONLY happen in the footer or not, and doesn't duplicate "update date".